infra/src/main/kotlin/fi/fta/geoviite/infra/publication/Publication.kt

```diff
@@ -398,6 +398,7 @@ data class ReferenceLinePublicationCandidate(
     override val publicationGroup: PublicationGroup? = null,
     override val cancelled: Boolean,
     val boundingBox: BoundingBox?,
+    var geometryChanges: List<Range<Double>>,
```

Comment:
Geometriamuutokset välitetään täällä rangeina m-arvoja. En ole varma saisiko tämän `val`:ifioitua kuinka helpolla, jää jatkotekemiseksi

Reply:
geometryChangesiahan ei päivitetä nähdäkseni missään :thinking: Eli valifionti taitaa olla sen kokoinen muutos, että kirjoittaa `var`in paikalle `val`.